pkg/api/types.go

```diff
@@ -145,9 +145,20 @@ func (r Resources) ValidateNonZero() error {
 	return nil
 }
 
+func (r Resources) SanityCheck() error {
+	if r.VCPU.Cmp(*resource.NewMilliQuantity(250, resource.BinarySI)) == -1 {
+		return errors.New("VCPU is too small")
+	}
+	if r.VCPU.Cmp(*resource.NewMilliQuantity(128*1000, resource.BinarySI)) == 1 {
```

Comment:
Same comment here about the bounds. 128 is maybe too small (again, maybe we want something bigger in the future). I'd use e.g. 512.

cc @tychoish for second opinion here.